packit_service_centosmsg/consumer.py

```diff
@@ -59,29 +58,38 @@ def celery_app(self):
         return self._celery_app
 
     def on_message(self, client, userdata, msg):
-        pprint(msg.topic)
-        pprint(json.loads(msg.payload))
-
+        logger.info(f"Received a message on topic: {msg.topic}")
+        subtopic = msg.topic.split("/", 1)[-1].split(".", 1)[0]
+        if self.subtopics and subtopic not in self.subtopics:
+            logger.debug(f"Ignore message: Subtopic {subtopic!r} not in {self.subtopics!r}.")
+            return
+        message = json.loads(msg.payload)
+        message["topic"] = msg.topic
+        logger.debug(json.dumps(message, indent=4))
         if os.getenv("SEND_CELERY_TASKS"):
-            msg.payload["topic"] = msg.topic
-            self.celery_app.send_task(
-                name="task.steve_jobs.process_message", kwargs={"event": msg.payload}
+            result = self.celery_app.send_task(
+                    name="task.steve_jobs.process_message", kwargs={"event": message, "source": "centosmsg"}
             )
-
-        logger.info(json.loads(msg.payload))
+            logger.info(f"Task UUID={result.id} sent to Celery.")
 
     def on_connect(self, client, userdata, flags, rc):
-        print("Connected with result code " + str(rc))
-        logger.info(f"Connected wit result code: ${str(rc)}")
-
+        logger.info(f"Connected wit result code: {rc}")
+        # TODO(csomh): try to make it a comma separated list of topics
+        topic = os.getenv("MQTT_TOPICS", "git.stg.centos.org/#")
+        # TODO(csomh): do we need a more flexible way to define sub-topics?
+        self.subtopics = list(filter(lambda x: bool(x), os.getenv("MQTT_SUBTOPICS", "").split(",")))
```

Comment:
just nitpicking :-)

```suggestion
        self.subtopics = [x for x in os.getenv("MQTT_SUBTOPICS", "").split(",") if bool(x)]
```

Reply:
That's what I call "over-engineered with style" :smile: 

Thanks for pointing out.